best-practices.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+    env: {
+        browser: true,
+        node: true
+    },
+    rules: {
+        // 'accessor-pairs': 2,
+        // 'array-callback-return': 2,
+        // 'block-scoped-var': 2,
+        // NO_RULE 'class-methods-use-this': 2,
```

Comment:
I don't really agree that we should add all the rules we don't use commented out. This makes it harder to read, and requires you to read the README to understand why are they commented out and to get what `NO_RULE` means. Also we have to maintain the commented out rules as well as ESLint is updating.

I would rather just remove these rules for now (the ones we don't use) and add them back 1 by 1 by PRs when we feel the need for them. Then the conversation on that specific rule can be tracked down in the PR itself.